README.md

```diff
@@ -22,9 +22,13 @@ Ensure that new API versions don't break API consumers with breaking change dete
 
 ![Breaking Change Detection](./media/breaking-change.png)
 
-[[Pre-Release]](#pre-release) Generate Open API specification file from API code.
+[[Pre-Release]](#pre-release) Use natural language to generate OpenAPI specification compliant with Spectral rulesets.

```

Comment:
Use GitHub Copilot for Azure to design API specifications that comply with API Center governance, ensuring your APIs adhere to best practices and standards through AI assistance.